### ISSUE TITLE
Version Management

### ISSUE THREAD

**Issue by kmbarlow** [jenv/jenv#386]
I am wondering if it would be possible to have an option to update the chain on the add algorithm.  When a new incremental version of java comes out and we add it (e.g. 17.0.6), if there is an existing mapping for the root (e.g 17 -> 17.0.2), the mapping will persist and not be updated to the newer 17.0.6.  

`jenv add /usr/local/Cellar/openjdk@17/17.0.6/
openjdk64-17.0.6 added
17.0.6 added
 17.0 already present, skip installation
 17.0.6 already present, skip installation`

I'd like an option to specify that the root should be updated so that they always point at latest of that release.

**Comment by stoerr** [comment]
Right, and if one is removing obsolete versions, the now broken links like 17 / 17.0 in the example aren't removed but still block the installation of the new JDK as those versions.
